python/tvm/micro/model_library_format.py

```diff
@@ -449,47 +498,53 @@ def _eval_shape(param_name, buffer_shape):
     return memory_map
 
 
-def _export_operator_model_library_format(mod: build_module.OperatorModule, tempdir):
+def _export_operator_model_library_format(
+    mods: typing.List[build_module.OperatorModule], tempdir: pathlib.Path
```

Comment:
i wonder if for operator MLF, we should just not allow multiple module export? if they don't have a mod_name, then we can't really identify them or namespace them properly. wdyt?

Reply:
yeah makes sense.